ci-operator/step-registry/windows/e2e/test/windows-e2e-test-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: windows-e2e-test
+  from: windows-machine-config-bootstrapper-tests
+  commands: windows-e2e-test-commands.sh
+  resources:
+   requests:
+      cpu: 10m
+      memory: 100Mi
```

Comment:
Does your test really use so little?

Reply:
increased the requests, really just kept the numbers from the configure step of e2e, which is not the same as running the test. Increased the requests